src/main/java/com/google/devtools/build/lib/query2/cquery/LabelAndConfigurationOutputFormatterCallback.java

```diff
@@ -18,39 +18,53 @@
 import com.google.devtools.build.lib.analysis.config.BuildConfiguration;
 import com.google.devtools.build.lib.analysis.config.CoreOptions.IncludeConfigFragmentsEnum;
 import com.google.devtools.build.lib.events.ExtendedEventHandler;
+import com.google.devtools.build.lib.packages.Target;
 import com.google.devtools.build.lib.query2.engine.QueryEnvironment.TargetAccessor;
 import com.google.devtools.build.lib.skyframe.SkyframeExecutor;
 import java.io.OutputStream;
 
 /** Default Output callback for cquery. Prints a label and configuration pair per result. */
 public class LabelAndConfigurationOutputFormatterCallback extends CqueryThreadsafeCallback {
-
+  private final boolean showKind;
   public LabelAndConfigurationOutputFormatterCallback(
       ExtendedEventHandler eventHandler,
       CqueryOptions options,
       OutputStream out,
       SkyframeExecutor skyframeExecutor,
-      TargetAccessor<ConfiguredTarget> accessor) {
-    super(eventHandler, options, out, skyframeExecutor, accessor);
+      TargetAccessor<ConfiguredTarget> accessor,
+      boolean showKind) {
+    super(eventHandler, options, out, skyframeExecutor, accessor, showKind);
```

Comment:
I don't think you need `showKind` on this line: since it's private to  `LabelAndConfigurationOutputFormatterCallback`, `CqueryThreadsafeCallback` doesn't know (or need to know) it exists.

But you do want to set the private final field again, i.e. `this.showKind = showKind`.